tests/test_datasets/test_dataset_functions.py

```diff
@@ -379,3 +380,112 @@ def test_create_dataset_numpy(self):
             paper_url=paper_url
         )
         dataset.publish()
+
+    def test_create_dataset_pandas(self):
+        # pandas is only a optional dependency and we need to skip the test if
+        # it is not installed.
```

Comment:
In the long run we will probably make pandas required (it is already required for the examples and documentation), so I'd totally okay if you add it as a dependency.

Reply:
This could be deleted now, right?